src/main/java/meteordevelopment/meteorclient/systems/modules/render/Tracers.java

```diff
@@ -372,4 +422,11 @@ private float getAlpha() {
     public String getInfoString() {
         return Integer.toString(count);
     }
+
+    public enum ListMode {
+        Whitelist,
+        Blacklist,
+        None,
```

Comment:
instead of having a `None` mode, make a boolean setting called "filter-items" or something which hides the three new settings when disabled